src/client/src/scenes/Editor/index.tsx

```diff
@@ -59,6 +59,8 @@ export const Editor: React.FC<IEditor> = inject('editorDataModel')(
             }
         };
 
+        console.log(extension);
+
```

Comment:
Will this console.log go to prod?

Reply:
@VoIdJMp Moving forward to advance other configs for pre-push trash code prevent.